connector/src/main/java/org/jboss/as/connector/subsystems/jca/JcaExtension.java

```diff
@@ -524,11 +528,16 @@ private void parseWorkManager(final XMLExtendedStreamReader reader, final ModelN
                     }
                     case ELYTRON_ENABLED: {
                         switch (readerNS) {
-                            case JCA_5_0: {
+                            case JCA_5_1: {
                                 String value = rawElementText(reader);
                                 JcaWorkManagerDefinition.WmParameters.ELYTRON_ENABLED.getAttribute().parseAndSetParameter(value, workManagerOperation, reader);
                                 break;
                             }
+                            case JCA_5_0: {
+                                String value = rawElementText(reader);
+                                JcaWorkManagerDefinition.WmParameters.ELYTRON_ENABLED_5_0.getAttribute().parseAndSetParameter(value, workManagerOperation, reader);
```

Comment:
This won't end up writing 'false' to the operation unless the xml says 'false'. The AttributeDefinition's default value doesn't factor into parseAndSetParameter.

To get behavior where not explicitly configuring elytron-enabled results in writing false to the operation if an old namespace is used, at the end of the method if it's an old namespace check if workManagerOperation.hasDefined(caWorkManagerDefinition.WmParameters.ELYTRON_ENABLED.getAttribute()) and if not add it set to false.

BTW, if you want you can drop support for namespaces prior to 5.0.

Reply:
fixed 

I will push the namespace drop in the next PR